src/Codebase/Infrastructure/Service/CodebaseService.php

```diff
@@ -36,4 +70,14 @@ public function parseSource(CodebaseRequestDto $codebaseRequestDto): CodebaseSou
     {
         return $this->sourceParser->parseSource($codebaseRequestDto);
     }
+
+    /**
+     * @param \Codebase\Application\Dto\ConfigurationRequestDto $configurationRequestDto
+     *
+     * @return \Codebase\Application\Dto\ConfigurationResponseDto
+     */
+    public function parseProjectConfiguration(ConfigurationRequestDto $configurationRequestDto): ConfigurationResponseDto
```

Comment:
Please describe the case when it will be useful to have this method as part of our contract?